### ISSUE TITLE
With default config, the webpack serve does not serve the images in html

### ISSUE THREAD

**Issue by mdtrooper** [webpack/webpack-cli#3990]
**Describe the bug**
I made a boiler plate with `npx webpack serve init .`, added a image as `<img .../>` in the default `index.html`. And the `$ npm run serve` does not show image in the browser.

**What is the current behavior?**

The browser returns 404 of image.

**To Reproduce**

```
$ cd /tmp/
$ mkdir foobar
$ cd foobar/
$ npm init -y
$ npm install -D webpack-cli
$ npx webpack-cli init .
? Which of the following JS solutions do you want to use? Typescript
? Do you want to use webpack-dev-server? Yes
? Do you want to simplify the creation of HTML files for your bundle? Yes
? Do you want to add PWA support? Yes
? Which of the following CSS solutions do you want to use? SASS
? Will you be using CSS styles along with SASS in your project? Yes
? Will you be using PostCSS in your project? No
? Do you want to extract CSS for every file? No
? Do you like to install prettier to format generated configuration? Yes
? Pick a package manager: npm

$ convert -size 500x500  plasma:fractal picture.jpg

# Edit index.html

$ cat index.html 
<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8" />
        <title>Webpack App</title>
    </head>
    <body>
        <h1>Hello world!</h1>
        <h2>Tip: Check your console</h2>
        <image src="picture.jpg" />
    </body>
    
    <script>
        if ("serviceWorker" in navigator) {
            window.addEventListener("load", () => {
                navigator.serviceWorker
                    .register("service-worker.js")
                    .then((registration) => {
                        console.log("Service Worker registered: ", registration);
                    })
                    .catch((registrationError) => {
                        console.error("Service Worker registration failed: ", registrationError);
                    });
            });
        }
    </script>
$ npm run serve
```

**Expected behavior**

The default config file must set a config to serve static images.

**Screenshots**

**Please paste the results of `npx webpack-cli info` here, and mention other relevant information**

```
$ npx webpack-cli info

  System:
    OS: Linux 6.2 Linux Mint 21 (Vanessa)
    CPU: (12) x64 AMD Ryzen 5 5500U with Radeon Graphics
    Memory: 4.32 GB / 22.30 GB
  Binaries:
    Node: 18.14.0 - ~/.nvm/versions/node/v18.14.0/bin/node
    Yarn: 1.22.19 - /usr/bin/yarn
    npm: 9.3.1 - ~/.nvm/versions/node/v18.14.0/bin/npm
  Browsers:
    Brave Browser: 118.1.59.124
    Chrome: 118.0.5993.117
  Packages:
    @webpack-cli/generators: ^3.0.7 => 3.0.7 
    css-loader: ^6.8.1 => 6.8.1 
    html-webpack-plugin: ^5.5.3 => 5.5.3 
    sass-loader: ^13.3.2 => 13.3.2 
    style-loader: ^3.3.1 => 3.3.1 
    ts-loader: ^9.5.0 => 9.5.0 
    webpack: ^5.89.0 => 5.89.0 
    webpack-cli: ^5.1.4 => 5.1.4 
    webpack-dev-server: ^4.15.1 => 4.15.1 
    workbox-webpack-plugin: ^6.5.4 => 6.5.4 
```

**Additional context**

 webpack.config.js 
```
// Generated using webpack-cli https://github.com/webpack/webpack-cli

const path = require('path');
const HtmlWebpackPlugin = require('html-webpack-plugin');
const WorkboxWebpackPlugin = require('workbox-webpack-plugin');

const isProduction = process.env.NODE_ENV == 'production';


const stylesHandler = 'style-loader';



const config = {
    entry: './src/index.ts',
    output: {
        path: path.resolve(__dirname, 'dist'),
    },
    devServer: {
        open: true,
        host: 'localhost',
    },
    plugins: [
        new HtmlWebpackPlugin({
            template: 'index.html',
        }),

        // Add your plugins here
        // Learn more about plugins from https://webpack.js.org/configuration/plugins/
    ],
    module: {
        rules: [
            {
                test: /\.(ts|tsx)$/i,
                loader: 'ts-loader',
                exclude: ['/node_modules/'],
            },
            {
                test: /\.css$/i,
                use: [stylesHandler,'css-loader'],
            },
            {
                test: /\.s[ac]ss$/i,
                use: [stylesHandler, 'css-loader', 'sass-loader'],
            },
            {
                test: /\.(eot|svg|ttf|woff|woff2|png|jpg|gif)$/i,
                type: 'asset',
            },

            // Add your rules for custom modules here
            // Learn more about loaders from https://webpack.js.org/loaders/
        ],
    },
    resolve: {
        extensions: ['.tsx', '.ts', '.jsx', '.js', '...'],
    },
};

module.exports = () => {
    if (isProduction) {
        config.mode = 'production';
        
        
        config.plugins.push(new WorkboxWebpackPlugin.GenerateSW());
        
    } else {
        config.mode = 'development';
    }
    return config;
};
```

```
$ npm -v
9.3.1

$ lsb_release --all
No LSB modules are available.
Distributor ID:	Linuxmint
Description:	Linux Mint 21
Release:	21
Codename:	vanessa

$ uname -a
Linux portatil 6.2.0-32-generic #32~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Fri Aug 18 10:40:13 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by mdtrooper** [comment]
Sorry....I labeled the issue as bug, I think that it is a "feature request" not a bug.

**Comment by evenstensberg** [comment]
Could you just create a reproduction repo?

**Comment by mdtrooper** [comment]
Yes, I upload it. 

[foobar.zip](https://github.com/webpack/webpack-cli/files/13377479/foobar.zip)

**Comment by evenstensberg** [comment]
Thx! Should be fixed in #4008

**Comment by mdtrooper** [comment]
I have one question (maybe offtopic, sorry) @evenstensberg .

I have clone your branch for testing if run fine the images in html. And I do `npm  install` and `npm run build` and well, I have the "compiled" packages:

```
$ ls packages/
configtest  generators  info  README.md  serve  webpack-cli
```

But I don't know how do this "local" packages....sorry, maybe It is a easy thing or first step for "npm devs" .
